AUDIT.md

```diff
@@ -0,0 +1,49 @@
+# Weave Security Audit Reports
+
+This file contains a brief summary of latest security audits applied to `Weave` by trusted third parties.
+
+## Scrutinized modules and aspects
+
+- General Conditions / Permission system design
+- Pieces related to money (and int overflow checks)
+- Input parsing and validations (buffer overflow, etc)
+- [x/sigs](./x/sigs)
+- [x/multisig](./x/multisig)
+- [x/gov](./x/gov)
+- [x/cash](./x/cash)
+- [x/distribution](./x/distribution)
+- [x/escrow](./x/escrow)
+
+## Report Summary
+
+In every audit, IOV is praised by high code quality standards.
+Observations of the audit firm on Weave:
+
+> Our general observations of the code concluded that the overall quality is very high and well structured. The design decision to develop the application layer in Go has had a great impact on security since the inherited security implications of the Go language makes it hard to write unsecure code. The design decision to use the well proven and heavily reviewed [Tendermint Consensus Engine](https://tendermint.com/) limits a lot of potential vulnerabilities. Since both the Tendermint core and Weave are written in the Go programming language, the integration code is minimal, and the integration interfaces are clean. This design decision in conjunction with the strict development environment that requires linters and security checks before building and committing any code has led to an overall code quality that is very good both from both a security as well as from a “code smell” hygiene and readability standpoint. Furthermore, the use of [gogo](https://github.com/gogo/protobuf), a protobuf implementation for network serialization, meant that many of the normal security problems (e.g buffer overruns and input validation problems) have been effectively eliminated. Weave only has 2 medium level possible security flaws and one of the flaws has been dismissed by our development team.
```

Comment:
My 0.02$, move the last sentence under the Possbile security vulnerabilities section.